1.9/administration/upgrading.md

```diff
@@ -31,11 +31,11 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
   - In 1.8 this can be found in the lower left corner of the DC/OS UI when screen is maximized.
   - In 1.9 this can be found under the Cluster menu.
 
-## Supported Upgrade Paths
+## Supported upgrade paths
 
-- 1.8.latest => 1.9.latest
-- 1.9.x => 1.9.(x+1)
-- 1.9.x => 1.9.x (for configuration changes)
+- From 1.8 stable to 1.9 stable.
```

Comment:
There's only one stable release at any given time, so this may be confusing. I think "latest" is the most accurate description of the upgrade path that we test. Before we release a new version of 1.9, we test upgrading from the latest version of 1.8.  Maybe instead of "latest" we can say "latest GA"?

> From the latest 1.8 GA release to the latest 1.9 GA release.